source/leuven/services/openondemand.rst

```diff
@@ -0,0 +1,250 @@
+.. _ood_t2_leuven:
+===========================================
+Open Ondemand on the KULeuven Tier2 cluster
+===========================================
+
+About
+-----
+
+Open OnDemand provides a user interface to HPC clusters from within a web browser. This tool supports a range of different apps and features that not only allow the
+user to easily submit jobs from within the browser session, but also provide different coding GUIs, tools for plotting and more.
+
+You can use this interface by navigating to the `KU Leuven Open Ondemand page`_. You can log in using your KU Leuven credentials. Access for vsc users from other 
+organizations is not yet supported. Once logged in, you'll notice you are connected with your VSC account. 
+
+Open Ondemand is only available for the Tier-2 wICE cluster.
+
+Use
+---
+
+The KU Leuven Open Ondemand page provides a range of functions:
+
+- Exploring, adapting and creating files
+- Submitting and monitoring jobs, creating job templates
+- Using a shell
+- Using interactive apps
+
+All of these functionalities can be used by accessing them through the tabs at the top of the page. In the following, we will describe the different parts in some more
+detail.
+
+Files
+-----
+
+This menu provides a file explorer that allows you to navigate files by clicking. You can access your ``$VSC_HOME`` and ``$VSC_DATA`` folders. Other storage is not
+available here. General file explorer options like moving, deleting, modifying and creating files or directories are available as well. You can also use this interface
+to download and upload files to and from your local machine. Be aware that this is not recommended for very large files.
+
+**Good to know:** the standard 'ctrl+s' sadly does not store your files on Open Ondemand, but will trigger a save on your local machine. Luckily, there is a quite 
+obvious save button on the editor page.
+
+Jobs
+----
+
+All jobs submitted from Open Ondemand will run on wICE. The Genius cluster is not supported. This also means that all your jobs should be submitted as **slurm** jobs.
+For more detail on how to run jobs on wICE, check out our :ref:`quickstart guide <wice_t2_leuven>`_.
+
+The jobs tab has two menus: 'Active Jobs' and 'Job Composer':
+
+Active jobs
+===========
+
+This lists all of your running, queued and completed jobs. Completed jobs will slowly dissapear after a couple of minutes. You have a handy overview of some job
+details by clicking the large arrow next to the job id, including things like the node list, the account you used, the status of the job... It also gives you the
+option to open the folder of the job script in the file manager. and thus inspect the created output and error files. 
+
+If your job is still running, you can also delete it by clicking the bin under 'Actions'. The 'Active jobs' menu does not allow re-submission of your job. How to
+(re-)submit jobs will be made clear in the next chapter.
+
+Job Composer
```

Comment:
Could you start this section with 1 or 2 sentences explaining why one might be interested in the Job Composer? I suppose that will be clear when reading the whole section, but it would be good to already summarize this upfront, i.e. describing what the Job Composer can be useful for.